packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/index.tsx

```diff
@@ -42,10 +43,22 @@ export const CustomTypeBuilder = () => {
     customTypeSM.tabs.find((tab) => tab.key === tabValue)?.sliceZone?.value
       .length === 0;
 
+  const tabWithUid = customTypeSM.tabs.find((tab) =>
+    tab.value.find((field) => field.key === "uid"),
+  );
+  const uidField = tabWithUid?.value.find((field) => field.key === "uid");
+
   return (
     <>
       <Window sx={sliceZoneEmpty ? { flexGrow: 1 } : undefined}>
-        {customType.format === "page" ? <WindowFrame /> : undefined}
+        {customType.format === "page" ? (
+          <WindowFrame
+            title={
```

Comment:
❓ #question

Here we may give `false` value no?

Reply:
If the value is `false` then the title is not displayed at all.
Is it a problem?

Reply:
I'm not a fan of passing `false` as a prop value just because ReactNode accept everything. I always like to be more clear and pass undefined. No strong opinion here.